src/features/projects/components/PopupProject.tsx

```diff
@@ -19,8 +19,8 @@ interface Props {
   open: boolean;
   handleOpen: Function;
   handleClose: Function;
-  buttonRef: any;
-  popupRef: any;
+  buttonRef: MutableRefObject<null>;
+  popupRef: MutableRefObject<null>;
```

Comment:
1. These seem to be the only properties of `Props` being actually used. Perhaps the rest could be removed?
2. The types for `buttonRef`/`popupRef` seems to be incomplete. Is the type of the ref that can be held always `null` or is that only when initialized in `Markers.tsx`?